zipline/pipeline/engine.py

```diff
@@ -96,24 +104,39 @@ class SimplePipelineEngine(object):
     asset_finder : zipline.assets.AssetFinder
         An AssetFinder instance.  We depend on the AssetFinder to determine
         which assets are in the top-level universe at any point in time.
+    populate_initial_workspace : callable, optional
+        A function which will be used to populate the initial workspace when
+        computing a pipeline. This function will be passed the
+        initial_workspace, the root mask term, the execution_plan, the dates
+        being computed for, and the assets requested and should return a new
```

Comment:
An easier way to document this might be to write a numpy-style docstring on `default_compute_initial_workspace`, and point the reader there for an example.